projects/algofi/index.js

```diff
@@ -278,7 +372,7 @@ module.exports = {
     algorand: {
         tvl: sdk.util.sumChainTvls([supply, dex]),
         borrowed,
-        staking
+        staking: stakingV1 + stakingV2
```

Comment:
this should be like tvl

`staking: sdk.util.sumChainTvls([stakingV1, stakingV2]),`

Reply:
done